packages/courier-js/README.md

```diff
@@ -1,45 +0,0 @@
-[![Courier: Your Complete Communication Stack](https://www.courier.com/_next/image/?url=https%3A%2F%2Fimages.ctfassets.net%2Fz7iqk1q8njt4%2F1PZo9WNTdmoDoYH3yulXa0%2Fb10830f7bfb09af5e644a39ac3d20c41%2FCourierJS_header_alt2.png&w=1920&q=75)](https://courier.com)
```

Comment:
Duplication of README here was explicit so that we can reflect the same on npm. Curious why we are deleting this

Reply:
re-added